filament/src/RenderPass.h

```diff
@@ -310,6 +311,10 @@ class RenderPass {
     // allocated commands ARE NOT freed, they're owned by the Arena
     ~RenderPass() noexcept;
 
+    // this must be called before calling getExecutor(), but can't be called from within
```

Comment:
nit: maybe this can be assert_invariant'd if you store a boolean? would help to avoid mistakes.